custom-connectors/PixelnebulaFlare/apiDefinition.swagger.json

```diff
@@ -0,0 +1,215 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Flare Digital Signage",
+    "description": "Digital signage software for professional results, in moments. Connect to Flare to manage your digital screens.",
+    "version": "1.0",
+    "contact": {
+      "name": "Flare Support",
+      "url": "https://pixelnebula.com/contact",
+      "email": "support@pixelnebula.com"
+    }
+  },
+  "host": "spark-dev.pixelnebula.com",

```

Comment:
@chrishatherall The host URL seems to be a test or dev domain URLs. Please note that we can not certify the connector which have test URLs.